ts/components/conversation/SessionConversation.tsx

```diff
@@ -221,7 +228,39 @@ export class SessionConversation extends React.Component<Props, State> {
       return <MessageView />;
     }
 
+    const conversation = getConversationController().get(selectedConversation.id);
```

Comment:
why don't you rely on `this.props.selectedConversation` directly?

Reply:
fingers moving faster than the brain